_hooks/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
OK, good to know.  I did also see that in the guidelines it is strongly preferred not to use ```.sh``` extension, so removed that from the travis-checks file.  Will try to follow that more in the future on any bash work.